conf/unique_conf/default_conf/developer.conf

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
This file should be ignored, it contains secrets (read about .gitignore) add this regex at unique_conf dir
and delete this file from history